libobs/graphics/graphics-ffmpeg.c

```diff
@@ -152,10 +152,12 @@ static void *ffmpeg_image_copy_data_straight(struct ffmpeg_image *info,
 	return data;
 }
 
-static inline size_t get_dst_position(size_t src OBS_UNUSED, const size_t w,
+static inline size_t get_dst_position(size_t src, const size_t w,
```

Comment:
Remove src parameter entirely?

Reply:
It does seem as though this was left in after adjustments on the initial PR, but it is not used at all in the function in its current form. I would agree that it's fine to remove.